ipn/ipnlocal/local.go

```diff
@@ -213,6 +213,7 @@ func (b *LocalBackend) UpdateStatus(sb *ipnstate.StatusBuilder) {
 	defer b.mu.Unlock()
 
 	sb.SetBackendState(b.state.String())
+	sb.SetAuthURL(b.authURL)
```

Comment:
out of paranoia, could you only set this if `b.state` is waiting-for-login, and set it to `""` in all other cases?

Reply:
As above, I don't want to start linking authURL to `state` given that it isn't anywhere else. If we decide to send more auth URLs for some reason, we shouldn't need to update the `tailscale status` code.

Reply:
So how's a user's supposed to use this?

You're presumably using this in Synology to do something, but when should a user go to this AuthURL. Documenting it as "oh, this is some field the control server sends sometimes.... meaning undefined, enjoy!" isn't particularly helpful.

But whatever. This isn't a stable API really. So submit away.

Reply:
My plan is to display it to the user whenever it's set along with a message "authenticate this client" or somesuch.

The problem is this is pretty undefined. I think current client behavior is to pop up an authURL in a browser under some (but not all?) conditions.